src/game/option.c

```diff
@@ -70,6 +78,121 @@ static TEXTSTRING *CtrlText[2] = { 0 };
 static TEXTSTRING *CtrlTextA[KEY_NUMBER_OF] = { 0 };
 static TEXTSTRING *CtrlTextB[KEY_NUMBER_OF] = { 0 };
 
+static const char *GetScanCodeName(int16_t key)
+{
+    // clang-format off
+    switch (key) {
+        case DIK_ESCAPE:       return "ESC";
+        case DIK_1:            return "1";
+        case DIK_2:            return "2";
+        case DIK_3:            return "3";
+        case DIK_4:            return "4";
+        case DIK_5:            return "5";
+        case DIK_6:            return "6";
+        case DIK_7:            return "7";
+        case DIK_8:            return "8";
+        case DIK_9:            return "9";
+        case DIK_0:            return "0";
+        case DIK_MINUS:        return "-";
+        case DIK_EQUALS:       return "+";
+        case DIK_BACK:         return "BKSP";
+        case DIK_TAB:          return "TAB";
+        case DIK_Q:            return "Q";
+        case DIK_W:            return "W";
+        case DIK_E:            return "E";
+        case DIK_R:            return "R";
+        case DIK_T:            return "T";
+        case DIK_Y:            return "Y";
+        case DIK_U:            return "U";
+        case DIK_I:            return "I";
+        case DIK_O:            return "O";
+        case DIK_P:            return "P";
+        case DIK_LBRACKET:     return "<";
```

Comment:
Microsoft says `DIK_LBRACKET` and `DIK_RBRACKET` are square brackets `[]`

Reply:
That's the value in the OG, and binding to `[` correctly displays `[`.